t/echo-udp.t

```diff
@@ -0,0 +1,146 @@
+# vi:set ft= ts=4 et:
+
+use Test::Nginx::Socket::Lua::Dgram;
+
+repeat_each(2);
+
+plan tests => repeat_each() * (blocks() * 4);
+
+run_tests;
+
+__DATA__
+
+=== TEST 1: simple single echo
+--- dgram_server_config
+echo "Hello, stream echo!";
+
+--- dgram_response
```

Comment:
Need a `--- dgram_request` section in these test cases otherwise the UDP server never gets a chance to run its request handler.